app/javascript/store.js

```diff
@@ -0,0 +1,7 @@
+import { configureStore, applyMiddleware } from '@reduxjs/toolkit';
```

Comment:
Please make sure that all dependencies used are listed in package.json file as @reduxjs/toolkit is missing

Reply:
- You are yet to list this dependency, as stated by the previous reviewer, in your package.json file. Please fix this 👍 👍 
```npm i @reduxjs/toolkit```